snmp/datadog_checks/snmp/data/profiles/_base_cisco_voice.yaml

```diff
@@ -0,0 +1,111 @@
+extends:
+  - _base_cisco.yaml
+
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    table:
+      name: hrSWRunPerfTable
+      OID: 1.3.6.1.2.1.25.5.1
+    symbols:
+      - name: hrSWRunPerfMem
+        OID: 1.3.6.1.2.1.25.5.1.1.2
+      - name: hrSWRunPerfCPU
+        OID: 1.3.6.1.2.1.25.5.1.1.1
+
+  - MIB: HOST-RESOURCES-MIB
+    name: hrSWRunStatus
+    OID: 1.3.6.1.2.1.25.4.2.1.7
```

Comment:
isn't this a table entry for `hrSWRunTable`? http://www.circitor.fr/Mibs/Html/H/HOST-RESOURCES-MIB.php#hrSWRunTable

Reply:
It could also be a useful cross table tag for the hrSWRunPerf metrics

Reply:
Yes, I must have missed that table!